### ISSUE TITLE
Parsing error when using nested values

### ISSUE THREAD

**Issue by PaulSonOfLars** [yoheimuta/protolint#108]
The following proto raises errors around the "additional_bindings" line, even though this is a valid proto ([another example here](https://github.com/grpc-ecosystem/grpc-gateway/blob/master/examples/proto/examplepb/flow_combination.proto#L58)).

```proto
syntax = "proto3";

package test;

message Test {
}

service RunnerService {
    rpc StreamTest (Test) returns (stream Test) {
        option (google.api.http) = {
            get: "/v1/messages/{message_id}"
            additional_bindings {
                get: "/v1/users/{user_id}/messages/{message_id}"
            }
        };
    }
}
```

I have also seen the following syntax be used instead, but this also raises an error. However, this one seems to be thrown in the Lexer, rather than the Parser (as determined from the error message constants). 
```proto
            additional_bindings: {
                get: "/v1/users/{user_id}/messages/{message_id}"
            };
```

PS: Sorry for the large amount of PRs/Issues raised 😅

**Comment by PaulSonOfLars** [comment]
Having had a good look at what the issue here is... [this](https://github.com/protocolbuffers/protobuf/issues/5075#issuecomment-415618391) has appeared quite relevant.
Turns out, the above is undocumented syntactic sugar for
```proto
        option (google.api.http).get = "/v1/messages/{message_id}";
        option (google.api.http).additional_bindings.get = "/v1/users/{user_id}/messages/{message_id}"; 
```
Which the protoc tool accepts just fine, but that noone deemed necessary to add to the spec.

**Comment by yoheimuta** [comment]
@PaulSonOfLars I published v0.22.1. Please confirm it works well.

- ref. https://github.com/yoheimuta/protolint/releases/tag/v0.22.1

**Comment by PaulSonOfLars** [comment]
Great, thank you! I will confirm tomorrow.

**Comment by PaulSonOfLars** [comment]
fixed in #110! thank you.
